cmd/kube-scheduler/app/options/deprecated.go

```diff
@@ -87,16 +87,14 @@ func (o *DeprecatedOptions) Validate() []error {
 	return errs
```

Comment:
Document in the applicable help strings above that scheduler would fail if the flags are used in combination with scheduling plugins.

Reply:
done